applications/konami/src/konami_code_fsm.erl

```diff
@@ -486,26 +486,40 @@ arm_bleg(#state{digit_timeout=Timeout}=State) ->
                 ,b_leg_armed='true'
                }.
 
+-spec maybe_fast_rearm(ne_binary(), ne_binary(), binary()) -> binary().
+-spec maybe_fast_rearm(ne_binary(), ne_binary(), binary(), boolean()) -> binary().
+
+maybe_fast_rearm(DTMF, BindingDigit, Collected) ->
+    maybe_fast_rearm(DTMF, BindingDigit, Collected
+        ,kapps_config:get_is_true(<<"metaflows">>, <<"use_fast_rearm">>, 'false')).
```

Comment:
I just finished testing this, but CONFIG_CAT from inside konami points to konami's config section, while most other settings are in the metaflows section.The config is most similar to the binding digit so I thought it should be in the same place https://github.com/2600hz/kazoo/blob/master/applications/konami/src/konami_config.erl#L64